File: LES_EXAMENS/EXAMEN_RATT_2023_2024/exercice_c.py
while True:
    n=int(input("saisir un entier n>0: "))
    if n>0 :
        break 
x=float(input("saisir un reel x : "))
sn=0
for i in range(1,n+1):
    sn+=i*(x**i)
print(f"sn={sn}")

File: LES_EXAMENS/EXAMEN_RATT_2023_2024/exercice_c.c
#include <stdio.h>
#include <math.h>
float sn,x;
int n;
int main(){
    while(1){
        printf("saisir un entier n>0: ");
        scanf("%d",&n);
        if(n>0)
            break;
    }
    sn=0;
    printf("saisir un reel x: ");
    scanf("%f",&x);
    for(int i=1;i<=n;i++){
        sn+=i*pow(x, i);
    }
    printf("sn=%f",sn);
    return 0;
}